src/components/admin/branchSettings/BranchFormModal.tsx

```diff
@@ -159,6 +173,31 @@ const BranchFormModal: React.FC<Props> = props => {
                   <Checkbox {...register("secret")}>Secret</Checkbox>
                 </FormControl>
               )}
+              {branchType === "APPLICATION" && (
+                <FormControl>
+                  <Checkbox {...register("automaticConfirmation.enabled")}>Enable Auto-Confirmation?</Checkbox>
+                </FormControl>
+              )}
+              {branchType === "APPLICATION" && automaticConfirmationEnabled && (
+                <FormControl isRequired>
+                  <FormLabel>Confirmation Branch</FormLabel>
+                  <Select {...register("automaticConfirmation.confirmationBranch.id")} placeholder="Select option">
+                    {
+                      props.branches && props.branches
+                      .filter((branch: Branch) => branch.type === BranchType.CONFIRMATION)
+                      .map((branch: Branch) => (
+                        <option value={branch.id}>{branch.name}</option>
+                      ))}
+                  </Select>
+                  <FormLabel>
+                    Emails
+                    <Tooltip hasArrow label={emailHelpMessage} bg='#d4d4d4' placement="right">
+                      <InfoIcon marginLeft={1} />
+                    </Tooltip>
+                  </FormLabel>
+                  <Input value={String(emails)} onChange={handleEmailInput} />
+                </FormControl>
```

Comment:
Each of these should be wrapped in their own <FormControl> I think you accidentally used one element for both of them.